### DISCUSSION TITLE
Taskbar clock customization - Date on lowercase

### DISCUSSION THREAD

**Discussion by SpectorM** [ramensoftware/windhawk-mods#225] [Q&A]
Greetings all!

When using the Taskbar clock customization mod, my date format changes to all lowercase (image below). Instead of 1st capital letter.

Any way to change it? Been playing around with Microsoft patterns but can't make it work properly.

Thanks in advance.

![image](https://github.com/ramensoftware/windhawk-mods/assets/134311734/2c0bb010-2a82-4c5e-ba19-7ed971663f9a)

**Comment by m417z** [comment]
That's the behavior of Windows for Portuguese, and I don't think there's a Windows option to change it.
Relevant posts that I found:

* https://superuser.com/questions/1675933/set-windows-10-long-date-with-capitalized-month-word
* https://stackoverflow.com/questions/74963064/how-to-capitalize-the-first-letter-of-months-and-weekdays-in-flutter-built-in-wi

What you can do is fork the mod and add logic in C++ that capitalizes the words. Let me know if you'd like to go that route and need help. The downside is that you'll have to keep the custom code in sync with future mod updates.

**Reply by SpectorM** [reply]
Hey bud, thanks for the reply.

Got it. Although it's just an aesthetic issue, it still disturbs me somehow. 

It's been several years since I coded something. Gonna check out the links you sent and see if I'm still able to do it. :)

Cheers!

**Reply by m417z** [reply]
You can add your code here and modify the `g_dateFormatted` string however you please:
https://github.com/ramensoftware/windhawk-mods/blob/9c814a48137a68e5e1262b98d84be86b08ab96e9/mods/taskbar-clock-customization.wh.cpp#L710C1
